src/widgets/buttons.ts

```diff
@@ -487,7 +533,7 @@ export function deleteButtonWithCheck (
  *
  * @returns <dDomElement> - the button
  */
-export function button (dom: HTMLDocument, iconURI: string | undefined, text: string, handler: (event: any) => void) {
+export function button (dom: HTMLDocument, iconURI: string | undefined, text: string, handler: (event: any) => void, options: ButtonWidgetOptions = { buttonType: ButtonType.Primary, filled: true }) {
```

Comment:
See above, for readability, try to use option variables in such a way that false is the default.